### DISCUSSION TITLE
Columns in laptop standard resolution

### DISCUSSION THREAD

**Discussion by shajithomas32** [KevinVandy/material-react-table#952] [General]
I have like 10 columns , some of the column header text are long , I don't use pagination. What's the best practice to have all columns displayed in all resolutions, even with a laptop it needs to display all columns in standard resolution..if columns resize when I shrink browser will be great . I tried all layout , issue I noticed is last columns comes with horizontal scroll bar in laptops. I just  want the best approach  for 10 columns fully fit to screen in a laptop 

  let columns = [];    
      columns.push({ header: '1', accessorKey:  '1', minSize: 40, maxSize: 1000, size: 140 });
      columns.push({header: '2', accessorKey:  '2', minSize: 40, maxSize: 1000, size: 170 });
      columns.push({header: ' 3    ', accessorKey:  '3', minSize: 40, maxSize: 1000, size: 170 });
     

      const table = useMaterialReactTable({
        columns,
        data, //data must be memoized or stable (useState, useMemo, defined outside of this component, etc.)
        enablePagination: false,
        enableBottomToolbar: false,
        enableColumnResizing: true,
        enableGrouping: false,
        enableStickyHeader: false,
        enableStickyFooter: false,
        enableFullScreenToggle: false,
        enableRowSelection: false,
        enableMultiRowSelection: false, //use radio buttons instead of checkboxes
        enableColumnActions: false,
        enableColumnFilters: false,

**Comment by shajithomas32** [comment]
Can somebody suggest a sample code to  have Columns  fit in a  laptop standard resolution, also  do same in big screen.
All columns to adjust and fit in all resolutions without leaving empty space

**Reply by KevinVandy** [reply]
https://www.material-react-table.com/docs/guides/column-size#but-i-want-my-columns-to-have-a-percentage-width

**Reply by shajithomas32** [reply]
thanks Kevin i will  follow it, as always thanks for your fast response, you have a blessed day

**Reply by shajithomas32** [reply]
Yes your solution worked well  for me ,you are very helpful
